odk1-src/aggregate-vm.rst

```diff
@@ -1,4 +1,65 @@
-ODK Aggregate Virtual Machine 
-=================================
+.. spelling::
 
-The `ODK Aggregate VM <https://gumroad.com/l/odk-aggregate-vm>`_ is a fully-configured install of Aggregate that you can run on any computer. It requires very little setup, works well without Internet connectivity, and gives you complete control over your data collection campaign.
+  readme
+  VirtualBox
+
+ODK Aggregate Virtual Machine
+=============================
+
+This document provides instructions on setting up ODK Aggregate using the `ODK Aggregate VM OVA file <https://github.com/opendatakit/aggregate/releases/download/v1.5.0-beta.0/ODK-Aggregate-v1.5.0-beta.0.ova.zip>`_ and `VirtualBox <https://www.virtualbox.org>`_.
+
+.. admonition:: Before you get started…
+
+  - Please read this entire document before installing the virtual machine (VM). Doing so will save you a lot of time and effort.
+
+Basic Usage
+-----------
+
+1. Download and install VirtualBox from `virtualbox.org <https://www.virtualbox.org>`_.
+2. Double-click `ODK-Aggregate-v1.5.0-beta.0.ova <https://github.com/opendatakit/aggregate/releases/download/v1.5.0-beta.0/ODK-Aggregate-v1.5.0-beta.0.ova.zip>`_ to import the VM. Accept the default settings.
```

Comment:
These instructions are unclear.

I suspect one would need to download the file first? And then open the .ova file?

(At present, it almost reads like double clicking on the link in the document will import the VM.)

It also is unclear what is meant by "Import" here. Will opening the .ova file launch a download? In installation Wizard? Something else?

Also --- will this link always be up to date? It looks like it contains version number info.
It would be better to link to a page where one can get the download.

Reply:
We can't upload an OVA to Github, so they need to download the zip, unzip it, and double-click the OVA. That will launch Virtualbox and get open up the Importer.

Agreed that we shouldn't use the direct version link. I would say something like "Download the latest ODK Aggregate OVA from the [Downloads](https://opendatakit.org/downloads/) page".

Reply:
We've been putting *Download the Installer* in "Before you begin" in other similar procedures. 

See, for example: https://docs.opendatakit.org/aggregate-app-engine/

Reply:
I think that using the word "installer" would not be precise because an OVA file is more like a .docx Word document, which you can open with some program